src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -290,6 +292,10 @@ object DemuxFastqs {
       |To output with Illumina conventions that match `bcl2fastq` and `BCLconvert`, use:
       |
       |`--fastq-skip-read-numbers=true --fastq-include-sample-barcodes=true --illumina-file-names=true`
+      |
+      |To output reads with a passing filter flag, use:
+      |
+      |`--keep-only-passing=true`
```

Comment:
needs more description of what a "passing filter flag" is and where it comes from (i.e. the `<is filtered>` field in the FASTQ comment above).